# Type of the requested sensor (required).
string sensor_type

# Name of the ROS package where sensor is located. (optional)
# If the package name is left undefined
# then the first known entry under the specified type is executed
string package_name

# name of the executable (optional)
# When undefined, first defined entry is used.
string executable

---

# name of the package
string package_name

# name of the executable/launchable/runnable
string executable

# return the name of the topic
string topic

# return code
#		0: Success
#		1: Failure
#	 -1: Undefined
int64 code

# return message
string message
